test/integration/admin/api/registry/policies_controller_test.rb

```diff
@@ -21,14 +21,44 @@ def setup
 
   test 'POST create responds with an error message when it is incorrect' do
     FactoryBot.create(:policy, policy_params[:policy].merge(account: @provider))
-    assert_no_difference(@provider.policies.method(:count)) do
+    assert_no_difference(Policy.method(:count)) do
       post admin_api_registry_policies_path(policy_params)
     end
     assert_response :unprocessable_entity
     assert_equal ['has already been taken'], JSON.parse(response.body).dig('errors', 'version')
   end
 
-  def policy_params
-    { policy: {name: 'my-name', version: 'my-version', schema: '{"foo": "bar"}'}, access_token: @access_token.value }
+  test 'POST create verifies permission' do
+    token_admin_with_wrong_scope = FactoryBot.create(:access_token, owner: @provider.admin_users.first!, scopes: %w[account_management], permission: 'rw').value
```

Comment:
Is it possible to only set the wrong values? Meaning, if it's a token admin with the wrong scope, declare as:
```
FactoryBot.create(:access_token, scopes: %w[account_management]).value
```
It's just a little bit cleaner.

Reply:
No, with `FactoryBot :access_token` it's not entirely possible what you want. If we omit `owner`, a different user of a different account will be build and it will be completely unrelated to `@provider`. The permission can be omitted though.